foursight_core/react/api/cognito.py

```diff
@@ -56,13 +70,14 @@ def _get_cognito_oauth_config_basic() -> dict:
     if not client_id:
         client_id = Gac.get_secret_value("COGNITO_CLIENT_ID")
     return {
-        "region": "us-east-1",
+        "region": AWS_REGION,
         "domain": domain,
-        "user_pool_id": user_pool_id,
-        "client_id": client_id,
+        "userpool": user_pool_id,
+        "client": client_id,
```

Comment:
Change to make naming convention (for /cognito_config endpoint) closer to what's there for the corresponding /auth0_config endpoint.

Reply:
Maybe I'm misunderstanding, and i might be overkill, but if this is just renaming things slightly from what Auth0 did, I point out that you could make this data-driven, making these be methods on a class that inherited from a common base, so that in a shared method, it returned
```
{"region" ..., "domain": ..., self.USERPOOL_ID: user_pool_id, self.CLIENT_ID: client_id, ...}
```
and then set USERPOOL_ID and CLIENT_ID as class variables of CognitoHandler differently than for Auth0Handler (or whatever you call the classes).

Just thinking aloud. Probably not needed.

Reply:
Yeah, at this point, the Auth0 is totally separate code, it's just the output JSON naming convention I was making similar.